wgpu-hal/src/gles/mod.rs

```diff
@@ -379,6 +380,43 @@ impl Texture {
             wgt::TextureDimension::D3 => glow::TEXTURE_3D,
         }
     }
+
+    /// More information can be found in issues #1614 and #1574
+    fn validate_target_heuristics(view_dimension: wgt::TextureViewDimension, target: u32) {
```

Comment:
We use "validate" a lot in this code base to mean spec validation checks which must pass to continue executing the function.
So to avoid confusion let's rename this into `log_failing_target_heuristics`.